YugaByte/yugabyte-db
# ENG-5293 #2024 [YSQL][docs] Unique constraint in CREATE and ALTER TABLE

Add UNIQUE constraint onto `latest/api/ysql/commands/ddl_create_table` and `latest/api/ysql/commands/ddl_alter_table` pages.
EBNF description is taken from Postgres's official doc

Test plan:
Manual

- start docs web browser
- check diagrams on `latest/api/ysql/commands/ddl_create_table` and `latest/api/ysql/commands/ddl_alter_table` pages
